ios/Exponent/ExponentView/ExponentViewManager.m

```diff
@@ -115,6 +116,12 @@ - (void)application:(UIApplication *)application didReceiveRemoteNotification:(N
   [[EXRemoteNotificationManager sharedInstance] handleRemoteNotification:notification fromBackground:isFromBackground];
```

Comment:
There is a null check inside `handleRemoteNotification:` so it won't pass anything to the experience. I think the `badgeNumber > 0` case is something I wrote in case we wanted to manually clear the badge number, but we don't currently use it. I will think about the best solution now that we have two handlers here.